h2o-r/tests/testdir_munging/runit_pubdev_5990_arrange_OOM.R

```diff
@@ -0,0 +1,17 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+test_arrange_OOM<- function(){
```

Comment:
I think we can drop either the Python test or the R test @wendycwong . It does the same thing and only prolongs the time it takes to finish the test suite.

Reply:
Removed the R test.  Thanks Pavel